modules/apps/commerce/commerce-checkout-web/src/main/resources/META-INF/resources/checkout_step/order_summary.jsp

```diff
@@ -421,6 +422,19 @@ Map<Long, List<CommerceOrderValidatorResult>> commerceOrderValidatorResultsMap =
 
 					<%= HtmlUtil.escape(shippingAddress.getCity()) %> <br />
 
+					<c:if test="<%= Validator.isNotNull(shippingAddress.getZip()) && checkoutDisplayContext.isOrderSummaryShowFullAddressEnabled() %>">
+						<%= HtmlUtil.escape(shippingAddress.getZip()) %> <br />
+					</c:if>
+
+					<c:if test="<%= (shippingAddress.getRegion() != null) && checkoutDisplayContext.isOrderSummaryShowFullAddressEnabled() %>">
+
+						<%
+						Region shippingRegion = shippingAddress.getRegion();
```

Comment:
I see that there is also a Country variable that is used for shipping and billing without conflicts. Can we try to call this only region?